recipes/default.rb

```diff
@@ -8,13 +8,27 @@
 include_recipe 'apt'
 include_recipe 'java'
 
-link "/etc/alternatives/java" do
-  to "/usr/lib/jvm/java-7-openjdk-amd64/bin/java"
-end
+#link "/etc/alternatives/java" do
+  #to "/usr/lib/jvm/java-7-openjdk-amd64/bin/java"
+#end
+
+#link "/etc/alternatives/javac" do
+  #to "/usr/lib/jvm/java-7-openjdk-amd64/bin/javac"
+#end
+
+include_recipe 'build-essential'
 
-link "/etc/alternatives/javac" do
-  to "/usr/lib/jvm/java-7-openjdk-amd64/bin/javac"
+case node['platform']
+when 'debian', 'ubuntu'
+  package 'ruby-dev' do
```

Comment:
Why do `ruby-dev` and `build_essential` need to be included here? I don't mind using/including them, but it I'm not entirely sure what role they are playing.